Shopify/cli
# Urls not fetched from API Client config

### WHY are these changes introduced?
Versioned app configuration has been 100% rolled out altogether with the 3.55.0 release. That means that the CLI should not use `api client configuration` anymore




### WHAT is this pull request doing?
- Remove `applicationUrl`  and `redirectUrlWhitelist` from any `app` API query
- Remove `GetUrls` query
- Use `applicationUrl`, `redirectUrlWhitelist` and `appProxy` urls from the current app version configuration where required
- `UpdateUrls` mutation is still being used because we don't want `dev` urls modifications to create a new app version but modify `API Client configuration` only


### How to test your changes?
- Create a `remix` app. Urls only affects to apps with `backend` and/or `frontend`
- `deploy` your app with some urls
- Run `dev` (remove `automatically_update_urls_on_dev` from the `toml` before if it's set). You should see the current app version urls when prompted to update them



### Post-release steps



### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
